stdlib/public/TensorFlow/Ops.swift

```diff
@@ -372,6 +372,23 @@ public extension Tensor where Scalar : Numeric {
   }
 }
 
+/// Performs batched matrix multiplication with another tensor and produces the
+/// result.
+@inlinable @inline(__always)
+@differentiable(
+  wrt: (lhs, rhs),
+  vjp: _vjpBatchedMatmul
+  where Scalar : Differentiable & FloatingPoint
+)
+public func batchedMatmul<Scalar : Numeric>(
+  _ lhs: Tensor<Scalar>,
+  _ rhs: Tensor<Scalar>,
+  adjointLHS: Bool = false,
```

Comment:
```suggestion
  adjointLeft: Bool = false,
```

Reply:
Guessing you also want the unlabeled `lhs` and `rhs` arguments to become `left` and `right`?

Reply:
Yeah. Rationale: I've never seen a precedent using `lhs`/`rhs` as argument labels, but I've seen `left`/`right`.